tests/net/socket/af_packet_ipproto_raw/src/main.c

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(net_test, CONFIG_NET_SOCKETS_LOG_LEVEL);
+
+#include <zephyr.h>
+#include <linker/sections.h>
+#include <ztest.h>
+#include <random/rand32.h>
+
+#include <fcntl.h>
+
+#include <net/ethernet.h>
+#include <net/dummy.h>
+#include <net/net_if.h>
+#include <net/socket.h>
+
+#if defined(CONFIG_NET_SOCKETS_LOG_LEVEL_DBG)
+#define DBG(fmt, ...) NET_DBG(fmt, ##__VA_ARGS__)
```

Comment:
Actually, I'd use NET_DBG directly instead of redefining a macro, IMO it's unnecessary complexity. If the DBG log level is disabled, logs won't be printed anyway.